linkis-engineconn-plugins/spark/src/main/java/org/apache/linkis/engineplugin/spark/client/deployment/KubernetesOperatorClusterDescriptorAdapter.java

```diff
@@ -230,4 +230,16 @@ public static SparkApplication getSparkApplication(String sparkOperatorName, Str
     sparkApplication.setMetadata(metadata);
     return sparkApplication;
   }
+
```

Comment:
1. Modify `getSparkApplicationClient` return value type to `MixedOperation<SparkApplication, SparkApplicationList, Resource<SparkApplication>>`
2. Suggest using the following method to monitor status 
```
getSparkApplicationClient(client).inNamespace(this.sparkConfig.getK8sNamespace()).withName(this.sparkConfig.getAppName()).watch(new Watcher<SparkApplication>() {
      @Override
      public void eventReceived(Action action, SparkApplication sparkApplication) {
        // todo get status
      }

      @Override
      public void onClose(WatcherException e) {

      }
    });
```

Reply:
Ok, thanks for reviewing the code